packages/react-native-hid/src/index.js

```diff
@@ -9,6 +9,34 @@ type DeviceObj = {
 };
 
 export default class HIDTransport extends Transport<DeviceObj> {
+  static observer: *;
+  static unsubscribed: boolean;
```

Comment:
I would prefer these are not static in the class but private global variables. So they are not exposed outside.

Reply:
ok